tests/secret_to_ckks/ops.mlir

```diff
@@ -0,0 +1,46 @@
+// RUN: heir-opt --canonicalize --secret-to-ckks %s | FileCheck %s
+
+!eui1 = !secret.secret<tensor<1024xi1>>
+!efi1 = !secret.secret<tensor<1024xf32>>
+
+module {
+  // CHECK-LABEL: func @test_arith_ops
+  func.func @test_arith_ops(%arg0 : !eui1, %arg1 : !eui1, %arg2 : !eui1) -> (!eui1) {
+    %0 = secret.generic ins(%arg0, %arg1 :  !eui1, !eui1) {
+    // CHECK: ckks.add
+      ^bb0(%ARG0 : tensor<1024xi1>, %ARG1 : tensor<1024xi1>):
+        %1 = arith.addi %ARG0, %ARG1 : tensor<1024xi1>
+        secret.yield %1 : tensor<1024xi1>
+    } -> !eui1
+    // CHECK: ckks.mul
+    // CHECK-NEXT: ckks.relinearize
```

Comment:
@lawrencekhlim we actually need a `ckks.rescale` here too to transform the scaling factor from delta^2 back down to delta, right?

Reply:
OK, it looks like BGV pipeline does actually translate Muls to MulNoRelin + Relin, so I'll include a Rescale operation as well in this IR - I think `heir-translate` on OpenFHE IR should take in options for auto / manual rescale and we can omit rescales after mults when it's auto.

Reply:
OK Now I go back on this - BGV pipeline does not add in ModulusSwitching, so I won't add CKKS Rescaling until we handle both (likely when we can define scaling factors and chains of moduli). For now we also translate Muls to Mul + Relin like BGV.